src/main/java/hudson/plugins/logparser/LogParserPublisher.java

```diff
@@ -206,9 +240,36 @@ public boolean configure(final StaplerRequest req, final JSONObject json)
                     "log-parser.").toArray(new ParserRuleFile[0]);
             useLegacyFormatting = json.getJSONObject("log-parser").getBoolean(
                     "useLegacyFormatting");
+            JSONObject colors = json.optJSONObject("log-parser");
+            if (colors != null) {
+                colorWarning = colors.optString("colorWarning");
```

Comment:
No it won't, basically the plugin creates a span with a style color: thecolor, so it won't change the color that's it.